frontend/src/components/controls/RisAlert.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import IconCheck from "~icons/ic/baseline-check"
+import IconErrorOutline from "~icons/ic/baseline-error-outline"
+import IconClose from "~icons/ic/baseline-close"
+
+defineProps<{
+  /**
+   * The variant of the alert
+   */
+  variant: "success" | "error"
+}>()
+
+defineEmits<{
+  /**
+   * Fired when the close button is clicked
+   */
+  close: []
+}>()
+</script>
+
+<template>
+  <div
+    role="alert"
+    class="flex gap-20 border-l-2 p-16"
+    :class="{
+      'border-l-green-700': variant === 'success',
```

Comment:
Nitpick: This could be simplified as `'border-l-green-700 bg-white': variant === 'success'` (same for the error)